.github/workflows/go.yml

```diff
@@ -84,6 +84,8 @@ jobs:
     - name: Run golangci-lint for netpol
       working-directory: hack/netpol
       run: make golangci
+    - name: Check copyright
+      run: ./ci/check-copyright.sh
```

Comment:
this should be a make target, not a shell script.

Reply:
@luolanzone  I think we need a check step here?  So we just need a shell script to generate git diff and return some err message like manifest? And we just check this in one step, so we don't need two shell scripts within one make target like test-tidy?

Reply:
I didn't mean there is unnecessary for check step, I mean you can add a check target in Makefile instead of a separate shell file considering you only have one parameter to check if it's check or not. I personally think it would be simpler if it's just a few lines.

Reply:
I don't think a Makefile target is strictly necessary (we don't have one for codegen, manifests). However I don't know why the CI step is part of the `golangci-lint` job and not part of the `tidy-codegen-manifest` job.

Reply:
yes, I talked with bangqi offline, after check the codes, I feel it's OK to keep them in a shell script.

Reply:
will move to the `tidy-codegen-manifest`, I think this job makes more sense